test/test_main.cpp

```diff
@@ -203,7 +203,7 @@ MeshGL WithIndexColors(const MeshGL& in) {
   out.runIndex.clear();
   out.runTransform.clear();
   out.faceID.clear();
-  out.runOriginalID = {Manifold::ReserveIDs(1)};
+  out.runOriginalID = {static_cast<uint32_t>(Manifold::ReserveIDs(1))};
```

Comment:
Hmm, I think I see now why you recommended keeping the IDs as uint32_t across the board. The casting is annoying - and *we* know the values are sequential and so down-casting is safe, but that's not a common expectation (people might think this is a GUID, and maybe it could be someday). Sorry for the flip-flop, but I think I like it better the first way now.